helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -95,6 +95,8 @@ splunkPlatform:
     #   num_seconds is the number of seconds to buffer in case of a backend outage
     #   requests_per_second is the average number of requests per seconds.
     queueSize: 5000
+    # When set, enables persistence and uses the component specified as a storage extension for the persistent queue
+    storage: true
```

Comment:
Given that we've been reproducing the exporter configuration layout here, I think we shouldn't override the same configuration with a different value type to avoid confusion. I mean that `storage` is a string pointing to a storage extension in the exporter interface, not a boolean flag. 

Maybe we add this interface instead:

```
splunkPlatform:
  persistentQueueEnabled:
    logs: false
    metrics: false
```

WDYT?

Reply:
good idea will do.